File: 58. Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        # for checking the string with no whitespaces
        if len(s.split(" "))==1: return len(s)
        # iterating string from reverse order
        ans=str()
        for i in range(len(s)-1,-1,-1):
            if s[i]!=" ":
                ans+=s[i]
            elif ans:
                return len(ans)
        # if string have no white space in begining
        return len(ans)

File: 58. Length of Last Word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let ans=0
    for (let i=(s.length-1);i>-1;i--){
        if (s[i]!=' '){
            ans+=1;
            continue;
        }
        else if (s[i]==' ' && ans){
            return ans;
        }
        continue;
    }
    return ans;
};
